WalletWasabi.Fluent/CrashReport/ViewModels/CrashReportWindowViewModel.cs

```diff
@@ -40,6 +40,8 @@ public CrashReportWindowViewModel(SerializableException serializedException)
 
 	public string Caption => $"A problem has occurred and Wasabi is unable to continue.";
 
+	public string Link => "https://github.com/zkSNACKs/WalletWasabi/issues";
```

Comment:
```suggestion
	public string Link => "https://github.com/zkSNACKs/WalletWasabi/issues/new?assignees=&labels=&template=bug-report.md";
```

Reply:
This suggestion makes sense, but `AboutViewModel.BugReportLink` should be updated. And everything should reference that to avoid duplications.